app/src/test/java/com/android/sample/model/receivers/NotificationReceiverTest.kt

```diff
@@ -1,92 +0,0 @@
-package com.android.sample.receivers
```

Comment:
I deleted this file because I noticed I made a mistake by duplicating it. This exact file is also in the android test for notification receiver tests, where it's supposed to be.